airbyte-cdk/python/airbyte_cdk/sources/file_based/availability_strategy/default_file_based_availability_strategy.py

```diff
@@ -21,7 +21,9 @@ class DefaultFileBasedAvailabilityStrategy(AbstractFileBasedAvailabilityStrategy
     def __init__(self, stream_reader: AbstractFileBasedStreamReader):
         self.stream_reader = stream_reader
 
-    def check_availability(self, stream: "AbstractFileBasedStream", logger: logging.Logger, _: Optional[Source]) -> Tuple[bool, Optional[str]]:  # type: ignore[override]
+    def check_availability(
```

Comment:
This will generate mypy error as `# type: ignore[override]` needs to be applied to the line where `stream: "AbstractFileBasedStream"` is

Reply:
Thanks! I'll check the other annotations touched by this diff as well.

Reply:
The others are OK, I checked with mypy using the config at `airbyte-cdk/python/mypy.ini`